CHANGELOG.md

```diff
@@ -20,6 +20,7 @@
 
 ### Features
 - Adds option to enable retries on errors encountered by the Snowflake connector ([#14](https://github.com/dbt-labs/dbt-snowflake/issues/14))
+- Adds tests for incremental model unique key parameter ([core #4618](https://github.com/dbt-labs/dbt-core/pull/4618))
```

Comment:
Can we move this to the `1.1.0` section? It looks like we haven't created that yet so could you add that and update the release date for `1.0.0` please ([it should be December 3rd](https://github.com/dbt-labs/dbt-snowflake/releases/tag/v1.0.0))?

Reply:
Certainly will do both here.